relay-event-schema/src/protocol/metrics_summary.rs

```diff
@@ -0,0 +1,33 @@
+#[cfg(feature = "jsonschema")]
+use relay_jsonschema_derive::JsonSchema;
+use relay_protocol::{Annotated, Array, Empty, FromValue, IntoValue, Object};
+
+use crate::processor::ProcessValue;
+
+/// A collection of [`MetricSummary`] items keyed by the metric.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct MetricsSummary(pub Object<Array<MetricSummary>>);
+
+/// The metric summary of a single metric that is emitted by the SDK.
+///
+/// The summary contains specific aggregate values that the metric had during the span's lifetime. A single span can
+/// have the same metric emitted multiple times, which is the reason for aggregates being computed in each summary.
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct MetricSummary {
```

Comment:
Is this only for distribution metrics? If yes, I think it'd be useful to add it in the docstrings.

Reply:
It seems to be for all metrics: https://github.com/getsentry/sentry-python/blob/master/tests/test_metrics.py#L599-L627